src/tests/settings/settings.test.js

```diff
@@ -0,0 +1,75 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../app';
+import {
+  superAdminToken,
+  createUsers,
+  newRoleData,
+  loggedInToken
+} from '../fixtures/users.fixture';
+
+chai.use(chaiHttp);
+chai.should();
+
+describe('Test for super admin reset user role:', () => {
+  before(async () => {
+    await createUsers();
+  });
+  it('It should allow Super-Admin user to reset user role', (done) => {
+    chai.request(app)
+      .patch('/api/users/settings/reset-roles')
+      .set('Authorization', superAdminToken, 'Accept', 'application/json')
+      .send(newRoleData)
+      .end((err, res) => {
+        res.body.should.be.an('object');
+        res.should.have.status(200);
+        res.body.should.have.property('message').equal('User role successfully updated');
+        res.body.should.have.property('data');
+        res.body.data.should.have.property('role').equal(`${newRoleData.userRole}`);
+        done();
+      });
+  });
+
+  it('It should NOT allow Non Super-Admin user to reset another user role', (done) => {
+    chai.request(app)
+      .patch('/api/users/settings/reset-roles')
+      .set('Authorization', loggedInToken, 'Accept', 'application/json')
+      .send(newRoleData)
+      .end((err, res) => {
+        res.body.should.be.an('object');
+        res.should.have.status(403);
+        res.body.should.have.property('message').equal('Only super admin can reset user role');
+        done();
+      });
+  });
+
+  it('It should NOT allow Super-Admin user to reset user role: User Not Found', (done) => {
+    const newEmail = 'paul@gmail.com';
```

Comment:
can you put this in separate file the call it here?

Reply:
I did it in victor's feedback